feat(ui): add footer to the website (#28)

AadityaBansal01 opened the pull request:
# Add footer

## Description
This PR adds a footer to the website (#28).

### Changes include:
- Added `Footer.tsx` component
- Integrated footer into `layout.tsx`
- Footer includes basic information and links
- Improves page completeness and overall design

NiranjanKumar001 reviewed (changes requested):
looking like vibe coding stuff ,make sure the design matches properly

AadityaBansal01 commented:
Hi, I’ve updated the footer to make the page more informative and visually consistent. Kindly review the PR and merge if everything looks good. Thank you!

AadityaBansal01 commented:
Hi, I’ve fixed the broken image reference and updated the footer layout to make it sleek and consistent. Kindly review and merge if everything looks good. Thank you!

NiranjanKumar001 commented:
not meeting criteria
